crates/rolldown_common/src/inner_bundler_options/mod.rs

```diff
@@ -76,7 +76,7 @@ pub struct BundlerOptions {
   #[cfg_attr(
     feature = "deserialize_bundler_options",
     serde(deserialize_with = "deserialize_treeshake", default),
-    schemars(with = "Option<bool>")
+    schemars(skip)
```

Comment:
If things still go wrong, just change the defnation to https://github.com/rolldown/rolldown/pull/1681#discussion_r1686188850 and everything should be ok to be auto `derived`.

Reply:
We have used the shchema to validate options

https://github.com/rolldown/rolldown/blob/2d2974e7b185b63cd7593093114e815e77b0d2a3/crates/rolldown_testing/src/test_config.rs#L39